From: Etienne Millon <etienne.millon@gmail.com>
Date: Sat, 29 Oct 2011 21:15:40 +0200
Subject: Change volume slider sensitivity

In previous versions, GMPC used a GtkVolumeButton which had a sensitivity of 2%.
Since 11.8.16, a homemade widget is used instead with a sensitivity of 5%.
This patch reverts to the previous behaviour.

Relevant Debian bug : #646822.
---
 src/Widgets/gmpc-progress.vala |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Widgets/gmpc-progress.vala b/src/Widgets/gmpc-progress.vala
index 3742374..f1082af 100644
--- a/src/Widgets/gmpc-progress.vala
+++ b/src/Widgets/gmpc-progress.vala
@@ -237,11 +237,11 @@ public class Gmpc.Progress : Gtk.VBox
     {
         if(event.direction == Gdk.ScrollDirection.UP)
         {
-            seek_event(this.current+5); 
+            seek_event(this.current+2);
         }
         else if (event.direction == Gdk.ScrollDirection.DOWN)
         {
-            seek_event(this.current-5); 
+            seek_event(this.current-2);
         }
         return false;
     }
